client/src/SessionsList.js

```diff
@@ -9,7 +9,7 @@ import {
   ListSubheader,
 } from '@material-ui/core'
 import { makeStyles } from '@material-ui/core/styles'
-import { Add, Done } from '@material-ui/icons'
+import { Add, EditOutlinedIcon } from '@material-ui/icons'
```

Comment:
material-ui/icons exports this icon as "EditOutlined"

if you want to name it "EditOutlinedIcon" you have to do `import { EditOutlined as EditOutlinedIcon }`
```suggestion
import { Add, EditOutlined as EditOutlinedIcon } from '@material-ui/icons'
```